site/content/en/docs/installation/_index.md

```diff
@@ -107,8 +107,8 @@ supports cert rotation), instead of internal cert, you can by performing the fol
 First, install cert-manager on your cluster by running the following command:
 
 ```shell
-VERSION=v1.11.0
-kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/$VERSION/cert-manager.yaml
+CERT_MANAGER_VERSION=v1.11.0
```

Comment:
nit: Cert manager's latest version is v1.17.0. Maybe we can update this as well?

Reply:
Thanks @ardaguclu 
It has been changed